src/GRA.Web/Areas/MissionControl/Views/Participants/Detail.cshtml

```diff
@@ -3,6 +3,9 @@
 @using GRA.Domain.Model
 @await Html.PartialAsync("_ParticipantPartial")
 
+
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
Localization isn't needed on the MissionControl side of things.

Reply:
Fixed in commit a644d762f46ed05c11f0f643e52b135003a8f354